app/views/course/assessment/assessments/_assessment_actions.json.jbuilder

```diff
@@ -2,7 +2,10 @@
 can_attempt = can?(:attempt, assessment)
 can_view_submissions = can?(:view_all_submissions, assessment)
 can_manage = can?(:manage, assessment)
+
 can_read_statistics = can?(:read_statistics, current_course)
+can_read_statistics &&= current_component_host[:course_statistics_component].present?
```

Comment:
Prefer setting in single line to follow convention everywhere else. We have never used `&&=` in our backend.  
```rb
can_read_statistics = current_component_host[:course_statistics_component].present? && can?(:read_statistics, current_course)
```
Also single line will set the `can_read_statistics` just once instead of setting twice, which in my opinion is cleaner in `.jbuilder`  
Semantically we should check if statistics component is enabled/present **before** checking if user is allowed to read stats.